src/ControlSurface/ScoreControlSurface.qml

```diff
@@ -31,26 +31,102 @@ import Variable.Global 1.0
 
 Column {
     id: controlSurfaceListColumn
-    spacing: 5
+
     property string name
 
-    Rectangle {
-        width: parent.width
-        height: 5
-        color: Skin.gray1
+    // Receiving information about controls in a control surface from score
+    Connections {
+        target: controlSurface
+
+        // Adding controls in a control surface
+        function onAppendControls(m) {
+            controlSurfaceName.text = m.Name
+            var i = 0
```

Comment:
in general prefer `let` (and `const` when it makes sense) to `var`: 
- https://dev.to/kunalt96/var-let-const-in-javascript-3dne
- https://medium.com/podiihq/javascript-variables-should-you-use-let-var-or-const-394f7645c88f
- https://babeljs.io/docs/en/learn#let--const